assets/scripts/evolution/minimalist_carousel_two.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Created by stu on 4/12/17.
+ */
+export default function () {
+    var carousel = document.getElementById("evo_c-carousel-border-nav");
+    var innerDefault = "evo_c-carousel-border-nav__inner--default";
+    var innerActive = "evo_c-carousel-border-nav__inner--active";
+    var borderActive = "evo_c-carousel-border-nav__border--active";
+    var borderDefault = "evo_c-carousel-border-nav__border--default";
+
+
+    carousel.addEventListener("click", function(e) {
```

Comment:
You will need to check if the `carousel` element exists in the DOM before doing anything else with it. You could use a simple `if` or something like:

```javascript
carousel && carousel.addEventListener(...)
```

This helps to avoid breaking the rest of the JS code in case an element was not found in the DOM.